samples/99_zil_interpreter/app/syntax.rb

```diff
@@ -94,4 +87,11 @@ def to_s
       "!#{@element}"
     end
   end
+
+  # #DECL (...)
+  class Decl < ElementWrapper
+    def to_s
+      "#DECL #{@elements}"
+    end
+  end
```

Comment:
Side note: I changed Decl to be an ElementWrapper rather than a "list" after all... since it's more convenient in the eval function....